lib/ex_doc/formatter/epub/templates.ex

```diff
@@ -68,4 +70,16 @@ defmodule ExDoc.Formatter.EPUB.Templates do
   EEx.function_from_file(:defp, :head_template,
                          Path.expand("templates/head_template.eex", __DIR__),
                          [:config, :page])
+
+  @media_types
```

Comment:
I think we could do the FIle.read! directly here as we don't use the module attribute anywhere.